OpenEdXMobile/src/main/java/org/edx/mobile/view/my_videos/MyRecentVideosFragment.java

```diff
@@ -77,9 +78,11 @@
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        environment.getAnalyticsRegistry().trackScreenView(Analytics.Screens.MY_VIDEOS_RECENT);
         setHasOptionsMenu(!isLandscape());
         EventBus.getDefault().register(this);
+        if (getUserVisibleHint()) {
```

Comment:
When we open the `My Videos` screen we need to send the screen event for the currently visible tab. Both of our fragments i.e. `Recent Videos` & `All Videos` are inside a `ViewPager`, the only reliable way to know which fragment is currently in the foreground is by calling this function i.e. `getUserVisibleHint`.